faker/providers/lorem/pl_PL/__init__.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import unicode_literals
```

Comment:
you'll need to add `# coding=utf-8` at the top of the file

Reply:
Fixed